packages/frontend/src/components/AboutFooter.tsx

```diff
@@ -0,0 +1,74 @@
+import {lightdashApi} from "../api";
+import {ApiError, ApiHealthResults, HealthState} from "common";
+import React, {useState} from "react";
+import {useQuery} from "react-query";
+import {AnchorButton, Callout, Classes, Dialog, H5, H6, Intent, Tag} from "@blueprintjs/core";
+
+const getHealthState = async () => {
+    return await lightdashApi<ApiHealthResults>({
+        url: `/health`,
+        method: 'GET',
+        body: undefined,
+    })
+}
+
+const AboutFooter = () => {
+    const [isOpen, setIsOpen] = useState(false);
+    const healthState = useQuery<HealthState, ApiError>({
```

Comment:
Too simple to move to it's own hook?

Reply:
Yes, this request is not going to be reused and the component is small enough to keep the code readable. Extracting it to a separate file/hook would only add boilerplate code.

Reply:
But we should use a query name enum. This avoids different places from using the same query name and caching results incorrectly.